pkg/asset/manifests/gcp/loadbalanceruid.go

```diff
@@ -0,0 +1,39 @@
+package gcp
+
+import (
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	gcptypes "github.com/openshift/installer/pkg/types/gcp"
+)
+
+const (
+	// UIDConfigMapName is the Key used to persist UIDs to configmaps.
+	UIDConfigMapName = "ingress-uid"
```

Comment:
why is this public?

Reply:
This came from the kube code (where it is public). Definitely doesn't need to be public here. Will fix